templates/emails/token_request.txt

```diff
@@ -0,0 +1,17 @@
+Dear REANA administrator,
+
+New user access token request:
```

Comment:
What do you think about adding the `reana_url` in the beginning?
```suggestion
New user access token request in <{{ reana_url }}>:
```

Reply:
Also, make it a phrase: 

```
There is a new access token request for http://reana-qa.cern.ch:
```